app/views/application/_mailer_doctype.html.haml

```diff
@@ -1 +0,0 @@
-<!DOCTYPE html PUBLIC "-//W3C//DTD HTML 4.01 Transitional//EN" "http://www.w3.org/TR/html4/loose.dtd">
```

Comment:
Yea.. My reading told me that there is very little difference between 4.0t and 5, and it was quite old. I like the simplicity of the 5.